Frends.FTP.UploadFiles/Frends.FTP.UploadFiles/Logging/TransferLogSink.cs

```diff
@@ -59,10 +60,13 @@ public void Emit(LogEvent logEvent)
             }
         }
 
+        // ExcludeFromCodeCoverage(Justification) is not supported for 4.7.1 and 2.0:
+        // This one is excluded because currently, it seems impossible to get past 79% coverage in CI, even though it's over 81% when run locally.
```

Comment:
This is not a valid reason for excluding this method from coverage test. You can create an object of this class and write a unit test which tests this method.